pipelines/docker-build/patch.yaml

```diff
@@ -22,10 +22,8 @@
     value: $(params.dockerfile)
   - name: CONTEXT
     value: $(params.path-context)
-  - name: BUILD_EXTRA_ARGS
-    value: "$(tasks.appstudio-configure-build.results.buildah-auth-param)"
-  - name: PUSH_EXTRA_ARGS
-    value: "$(tasks.appstudio-configure-build.results.buildah-auth-param)"
+  - name: DOCKER_AUTH
```

Comment:
I wonder if passing the auth info as a parameter will cause it to be recorded in plain text in the PipelineRun/TaskRun resource which may be added as plain text to the image attestation.

Reply:
this is path to the folder containing config.yaml, so it does not show the secret in the plaintext.

Reply:
beautiful!